### PR TITLE
Added more details on the timestamp placeholder for S3/GCS connectors.

### PR DESCRIPTION
**Pull request by tomasz-sadura**

### CODE REVIEW

**Review by bochenekmartin — APPROVED** [review]

**Review by micheleRP — APPROVED** [review]
lgtm

**Review by Deflaimun — CHANGES_REQUESTED** [review]
dont merge yet. There´s a problem with the backticks 
image

**Comment by Deflaimun** [comment]
I believe they need to be escaped because of the `|`  
fixing it now...

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc [hunk 1]
```diff
@@ -102,7 +102,7 @@ a| The template for file names on GCS. Supports `{{ variable }}` placeholders fo
 - `topic`
 - `partition`
 - `start_offset` (the offset of the first record in the file)
-- `timestamp`
+- `timestamp:unit=yyyy|MM|dd|HH` (the timestamp of the record)
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc]:
```suggestion
- `timestamp:unit=yyyy\|MM\|dd\|HH` (the timestamp of the record)
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc [hunk 2]
```diff
@@ -127,7 +127,7 @@ a| The template for file names on S3. Supports `{{ variable }}` placeholders for
 - `topic`
 - `partition`
 - `start_offset` (the offset of the first record in the file)
-- `timestamp`
+- `timestamp:unit=yyyy|MM|dd|HH` (the timestamp of the record)
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc]:
```suggestion
- `timestamp:unit=yyyy\|MM\|dd\|HH` (the timestamp of the record)
```

**Review by Deflaimun — APPROVED** [review]

**Pull request merged**